lib/BloqadeSchema/src/transform.jl

```diff
@@ -233,7 +235,40 @@ function clip_waveform(wf::Waveform{BloqadeWaveforms.PiecewiseLinear{T,I},T},nam
     )
 end
 
-function hardware_transform_Ω(Ω,device_capabilities::DeviceCapabilities)
+function clip_waveform(wf::Waveform{BloqadeWaveforms.PiecewiseConstant{T},T},name,min_value::T,max_value::T) where {T<:Real,I}
```

Comment:
```suggestion
function clip_waveform(wf::Waveform{BloqadeWaveforms.PiecewiseConstant{T},T},name,min_value::T,max_value::T) where {T<:Real}
```
The `I` parameter isn't being used, can be removed